### DISCUSSION TITLE
some questions :

### DISCUSSION THREAD

**Discussion by Manix80** [YaLTeR/niri#704] [Q&A]
Hello, for the waybar tooltips that remain in the background, is there a way to move them to the front.
I tried to put waybar in “overlay” mode but I don't have the tooltips at all.
https://github.com/YaLTeR/niri/wiki/Layer%E2%80%90Shell-Components

for discord i have the same problem as here https://github.com/YaLTeR/niri/issues/405 is there a way to launch it without changing packages?

**Comment by YaLTeR** [comment]
Put Waybar on the top layer, not overlay mode.

To run Discord, use xwayland-satellite: https://github.com/YaLTeR/niri/wiki/Xwayland

**Reply by Manix80** [reply]
Thank you for your quick response! everything works properly
